try:
    try:
        1
    except Exception:
        1
except Exception:
    1
